tests/ops/service/privacy_request/test_request_runner_service.py

```diff
@@ -2254,3 +2258,148 @@ def test_needs_batch_email_send_new_workflow(
+    items = {
+        "customer_identifier": {
+            "customer_id": {"S": customer_name},
+            "email": {"S": customer_email},
+            "name": {"S": customer_name},
+            "created": {"S": datetime.now(timezone.utc).isoformat()},
+        },
+        "customer": {
+            "id": {"S": customer_name},
+            "name": {"S": customer_name},
+            "email": {"S": customer_email},
+            "address_id": {"L": [{"S": customer_name}, {"S": customer_name}]},
+            "personal_info": {"M": {"gender": {"S": "male"}, "age": {"S": "99"}}},
+            "created": {"S": datetime.now(timezone.utc).isoformat()},
+        },
+        "address": {
+            "id": {"S": customer_name},
+            "city": {"S": "city"},
+            "house": {"S": "house"},
+            "state": {"S": "state"},
+            "street": {"S": "street"},
+            "zip": {"S": "zip"},
+        },
+    }
+
+    for table_name, item in items.items():
+        res = dynamodb_client.put_item(
+            TableName=table_name,
+            Item=item,
+        )
+        assert res["ResponseMetadata"]["HTTPStatusCode"] == 200
+
+    yield {
+        "email": customer_email,
+        "formatted_email": customer_email,
+        "name": customer_name,
+        "customer_id": uuid,
+        "client": dynamodb_client,
+    }
+    # Remove test data and close Dynamodb connection in teardown
+    delete_items = {
+        "customer_identifier": {"email": {"S": customer_email}},
+        "customer": {"id": {"S": customer_name}},
+        "address": {"id": {"S": customer_name}},
+    }
+    for table_name, item in delete_items.items():
+        res = dynamodb_client.delete_item(
+            TableName=table_name,
+            Key=item,
+        )
+        assert res["ResponseMetadata"]["HTTPStatusCode"] == 200
+
+
+@pytest.mark.integration_external
+@pytest.mark.integration_dynamodb
+def test_create_and_process_access_request_dynamodb(
+    dynamodb_resources,
+    db,
+    cache,
+    policy,
+    run_privacy_request_task,
+):
+    customer_email = dynamodb_resources["email"]
+    customer_name = dynamodb_resources["name"]
+    customer_id = dynamodb_resources["customer_id"]
+    data = {
+        "requested_at": "2021-08-30T16:09:37.359Z",
+        "policy_key": policy.key,
+        "identity": {"email": customer_email},
+    }
+
+    pr = get_privacy_request_results(
+        db,
+        policy,
+        run_privacy_request_task,
+        data,
+        task_timeout=PRIVACY_REQUEST_TASK_TIMEOUT_EXTERNAL,
+    )
+    results = pr.get_results()
+    customer_table_key = (
+        f"EN_{pr.id}__access_request__dynamodb_example_test_dataset:customer"
+    )
+    address_table_key = (
+        f"EN_{pr.id}__access_request__dynamodb_example_test_dataset:address"
+    )
+    assert len(results[customer_table_key]) == 1
+    assert len(results[address_table_key]) == 2
```

Comment:
Would be good to also assert what's in the address table

Reply:
Added assertion for the only value there in https://github.com/ethyca/fides/pull/2998/commits/2be77e4c8135ff543f5f5bb389157dbf52f8e656